_src/assets/js/collapsible_form.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
```

Comment:
I think that 'use strict' should only go to the main because when you put together it will be repeated.

Reply:
Thanks for the review! We test it and still working! ;)